python/import_util.py

```diff
@@ -0,0 +1,36 @@
+import mgp
+import json as js
+import sys
+
+
+@mgp.write_proc
+def json(ctx: mgp.ProcCtx, path: str) -> mgp.Record():
+    """
+    Procedure to import the local JSON created by the export_util.json procedure.
+
+    Parameters
+    ----------
+    path : str
+        Path to the JSON that is being imported.
+    """
+    file = open(path)
+    list = js.load(file)
+    vertex_ids = dict()
+
+    for el in list:
```

Comment:
When using variables please keep the names easier to understand: `el` -> `element`